client/src/Asset.tsx

```diff
@@ -116,11 +127,18 @@ function Asset(): JSX.Element {
     const classes = useStyles();
     const { assetId } = useParams<{ assetId?: string }>();
     const asset = useAsset(assetId);
-
     const [searchText, setSearchText] = React.useState<string>('');
     const [selectedImgInd, setSelectedImgInd] = React.useState<number>(0);
+    const [showSnackbar, setShowSnackbar] = React.useState<boolean>(false)
+
+
 
-    const handleClaim = () => {
+
+    const handleClaim = (e:any) => {
+
+        if(e.currentTarget.type==="button"){
+            setShowSnackbar(true)
+    }
```

Comment:
```
    const handleClaim = (e: React.HtmlButtonElement) => {
            setShowSnackbar(true);
    }
```
we shouldn't need the if, since the only place we put this handler function is on the button element
unless maybe you were trying to appease typescript? changing the arg type should do the trick